src/comical.ts

```diff
@@ -183,6 +184,127 @@ export default class Comical {
+  // the root element most recently configured for Comical with convertBubbleJsonToCanvas().
+  public static initializeChild(
+    childElement: HTMLElement,
+    parentElement: HTMLElement
+  ) {
+    const parentBubble = Comical.allBubbles.find(
+      x => x.content === parentElement
+    );
+    if (!parentBubble) {
+      console.error(
+        "trying to make child of element not already active in Comical"
+      );
+      return;
+    }
+    const parentSpec = parentBubble.getBubbleSpec();
+    let familyLevel = parentSpec.level;
+    if (!parentSpec.order) {
+      // It's important not to use zero for order, since that will be treated
+      // as an unspecified order.
+      parentSpec.order = 1;
+      parentBubble.persistBubbleSpec();
+    }
+    // enhance: if familyLevel is undefined, set it to a number one greater than
+    // any level that occurs in allBubbles.
+    let childBubble = Comical.allBubbles.find(x => x.content === childElement);
+    if (!childBubble) {
+      childBubble = new Bubble(childElement);
+    }
+    const family = Comical.allBubbles
+      .filter(
+        x => x.getBubbleSpec().level === familyLevel && x.getBubbleSpec().order
+      )
+      .sort(
+        (a, b) =>
+          (a.getBubbleSpec().order || -1) - (b.getBubbleSpec().order || -1)
+      );
+    // we set order on parentBubble, so there is at least one in the family.
+    const lastInFamily = family[family.length - 1];
+    const maxOrder = lastInFamily.getBubbleSpec().order || 1;
+    const tip = lastInFamily.calculateTailStartPoint();
+    const root = childBubble.calculateTailStartPoint();
+    const mid = Bubble.defaultMid(root, tip);
+    // We deliberately do NOT keep any properties the child bubble already has.
+    // Apart from the necessary properties for being a child, it will take
+    // all its properties from the parent.
+    const newBubbleSpec: BubbleSpec = {
+      version: Comical.bubbleVersion,
+      style: parentSpec.style,
+      tails: [
+        {
+          tipX: tip.x!,
+          tipY: tip.y!,
+          midpointX: mid.x!,
+          midpointY: mid.y!,
+          joiner: true
+        }
+      ],
+      level: parentSpec.level,
+      order: maxOrder + 1
+    };
+    childBubble.setBubbleSpec(newBubbleSpec);
+    // enhance: we could possibly do something here to make the appropriate
+    // shapes for childBubble and the tail that links it to the previous bubble.
+    // However, currently our only client always does a fresh convertBubbleJsonToCanvas
+    // after making a new child. That will automatically sort things out.
+    // Note that getting all the shapes updated properly could be nontrivial
+    // if childElement already has a bubble...it may need to change shape, lose tails,
+    // change other properties,...
+  }
+
+  public static findChild(bubble: Bubble): Bubble | undefined {
+    const familyLevel = bubble.getSpecLevel();
+    const familyOrder = bubble.getBubbleSpec().order || -1;
```

Comment:
I think this variable should be more like bubbleOrder since it's not a single value for the whole family.